javascript-modal/main.js

```diff
@@ -0,0 +1,24 @@
+var clickStatus = false;
+var dialogContainer = document.querySelector('.dialog-container');
+var openButton = document.querySelector('.open-button');
+var noButton = document.querySelector('.no-button');
+var overlay = document.querySelector('#overlay');
+
+function modalClicked(event) {
+  if (clickStatus === false) {
+    clickStatus = true;
```

Comment:
You can remove this variable and these conditionals. Each of these separate functions does their own job, and should always do the same work when that function is called. There is no need for a boolean flag in this situation.